### PR TITLE
Consider only UI-made changes in synchronization API

### PR DESCRIPTION
**Pull request by vincentvanbush**

### CODE REVIEW

**Comment by Bonias** [comment]
I'm wondering: we have `is_changed` flag at the moment. I think this flag means that translation was updated at web UI. If I'm right that we don't need another column. We can just use `is_changed` with `updated_at`.

**Comment by mlitwiniuk** [comment]
@Bonias made valid point here, I think it's worth trying out first and checking how it will work in real life, production app

**Comment by mlitwiniuk** [comment]
@vincentvanbush please create another PR, we'll leave this one open just in case.

**Comment by vincentvanbush** [comment]
Are localization records ever _updated_ from outside the UI? The `is_changed` field is updated in a `before_update` callback, hence - if that's the case - it will not work as we are expecting.

**Comment by mlitwiniuk** [comment]
@vincentvanbush ok, point taken. 

@Bonias it happenes sometimes, that when you try to synchronize recently changed translations, a lot of "changed" translations (from ie. simple_form) are fetched as well. Problem is, that those changes do not introduce anything to existing translations database, they are just empty records.

File: app/models/lit/localization.rb [hunk 1]
```diff
@@ -6,7 +6,9 @@ class Localization < ActiveRecord::Base
     ## SCOPES
     scope :changed, proc { where(is_changed: true) }
     # @HACK: dirty, find a way to round date to full second
-    scope :after, proc { |dt| where('updated_at >= ?', dt + 1.second) }
+    scope :after, proc { |dt|
```
Comment (mlitwiniuk) [inline app/models/lit/localization.rb]: scope with params should be rather changed into class methods.

**Comment by Bonias** [comment]
Maybe synchronization works that way right now, but if we change synchronization to get only updated records (using is_changed flag) then we will get only real changed translations. Other synchronizations won't affect on that.

To summarize, we have to methods of updating translations right now:
1. Using web UI - it change `is_changed` - it's fine
2. Using synchronization - which currently fetch all translations and wrongly updates `is_changed` flag, but it would work well if we change it to fetch only updated translations

BTW. we could update `update_is_changed_attribute` to update `is_changed` only if translation text really change.

**Comment by vincentvanbush** [comment]
Obsoleted by #74.

**Pull request closed without merging**